addons/survey/views/survey_templates.xml

```diff
@@ -208,15 +208,15 @@
         </form>
 
         <!-- Modal used to display error message, i.c.o. ajax error -->
-        <div class="modal fade" id="AJAXErrorModal" role="dialog" aria-labelledby="AJAXErrorModal" aria-hidden="true" >
+        <div role="dialog" class="modal fade" id="AJAXErrorModal" >
             <div class="modal-dialog">
                 <div class="modal-content">
-                    <div class="modal-header">
-                        <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&amp;times;</button>
+                    <header class="modal-header">
```

Comment:
why do you replace div by header? I get it that it might make more sense but I wonder if it has repercussions on the display of the modal.

Reply:
It has no repercussion on the display. It was tested.